python/plugins/processing/algs/gdal/ogr2ogrtopostgislist.py

```diff
@@ -364,6 +364,10 @@ def getConsoleCommands(self, parameters, context, feedback, executing=True):
         input_details = self.getOgrCompatibleSource(
             self.INPUT, parameters, context, feedback, executing
         )
+        if not input_details.layer_name:
```

Comment:
This algorithm looks tolerant to empty layer names, so let's be safe:
```suggestion
        if input_details.layer_name is None:
```